cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -128,6 +129,15 @@ func createStaticPodFiles(manifestDir string, cfg *kubeadmapi.MasterConfiguratio
 		return err
 	}
 
+	// use the default audit policy if no audit policy exists
+	auditPolicyFilePath := filepath.Join(cfg.AuditPolicyDir, kubeadmconstants.AuditPolicyFileName)
```

Comment:
Pinned file names inside user specified directories makes for a really opaque interface. Is there a reason not to simply have the user specify the policy file directly?

Reply:
I could add an `--audit-policy-file` flag to combine with the `--audit-policy-dir` flag to make this customizable but default to `audit.yaml`. Does that sound reasonable?

Reply:
why not just `--audit-policy-file` as the full filepath and drop `--audit-policy-dir`?

Reply:
that seems sensible! 👍 thanks for the input